roadmaps/android.md

```diff
@@ -38,20 +38,24 @@ This roadmap is a high level overview of planned features and fixes. See the
 spreadsheet](https://docs.google.com/spreadsheets/d/1-BjWLdv3SKGnEBOdLmzPPhTPNErRVMQerjrI60N_DVs/edit#gid=1123749242)
 for a list of finer-grained tasks.
 
-As of Q3 2018, the native Android rules are incrementally rewritten and migrated
-to [rules_android](https://github.com/bazelbuild/rules_android). With this, the
-development of the Android rules can be decoupled from the core build system
-development.
+As of Q3 2018, the native Android rules are being incrementally rewritten and
+migrated to [rules_android](https://github.com/bazelbuild/rules_android). With
+this, the development of the Android rules can be decoupled from the core build
+system development.
 
 ### Q3 2018
 
 #### Build performance
 
-- Publish a prototype continuous build performance dashboard for Android projects
+- Publish a prototype continuous build performance dashboard for Android projects.
 
 #### Developer experience
 
 - Document Android with Bazel compatibility across Windows, macOS and Linux.
+- Introduce a best-practices documentation on configuring your Android build.
+- Introduce `tools_android` repository for common Android build tools for use
+  with Bazel (e.g. Firebase, Google Play Services)
```

Comment:
in this case it's just "Google Services". "Google Play Services" is a different thing.